src/app/GitExtensions/Program.cs

```diff
@@ -67,23 +67,6 @@ private static void Main()
             AppSettings.SetDocumentationBaseUrl(AppSettings.ProductVersion);
 
             ThemeModule.Load();
-#if SUPPORT_THEME_HOOKS
-            Application.ApplicationExit += (s, e) => ThemeModule.Unload();
-
-            SystemEvents.UserPreferenceChanged += (s, e) =>
-            {
-                // Whenever a user changes monitor scaling (e.g. 100%->125%) unload and
-                // reload the theme, and repaint all forms
-                if (e.Category == UserPreferenceCategory.Desktop || e.Category == UserPreferenceCategory.VisualStyle)
-                {
-                    ThemeModule.ReloadWin32ThemeData();
-                    foreach (Form form in Application.OpenForms)
-                    {
-                        form.BeginInvoke((MethodInvoker)(() => form.Invalidate()));
-                    }
-                }
-            };
-#endif
 
             HighDpiMouseCursors.Enable();
```

Comment:
I was kind of expecting we'd be calling [`Application.SetColorMode API`](https://learn.microsoft.com/dotnet/api/system.windows.forms.application.setcolormode) on line 45 depending on the user selection:

```cs
Application.SetColorMode(SystemColorMode);
```

Reply:
I just added a TODO for the Dashboard, to be cleaned up. That works well enough for now.